gdbinit.py

```diff
@@ -168,7 +168,12 @@ def main() -> None:
             sys.stdout.flush()
             os._exit(1)
 
-        update_deps(src_root, venv_path)
+        # Check for the environment variable to disable auto-update
+        auto_update_disabled = os.getenv("PWNDBG_AUTO_UPDATE", "true").lower() == "false"
```

Comment:
Let's change this to `PWNDBG_NO_AUTOUPDATE` and if it is defined with any value, skip updating deps.

Reply:
Okay I will change it asap.